### PR TITLE
BUG: Add parameter which stores the requested number of work units.

### PR DESCRIPTION
**Pull request by hjmjohnson**
http://review.source.kitware.com/#/c/23864/

Bradley Lowekamp
Uploaded patch set 1.Nov 2 4:22 PM

Kitware Build Robot
Patch Set 1: Verified+1 Build Successful: CDash filtered results: https://open.cdash.org/index.php?&project=Insight&filtercount=3&field1=buildname/string&compare1=63&value1=23864-1&field2=buildstarttime/date&compare2=83&value2=2015-03-01&field3=buildstarttime/date&compare3=84&value3=2029-01-01Nov 2 5:30 PM

Hans J. Johnson
Patch Set 1: Code-Review+2Nov 3 2:02 PM

Bradley Lowekamp
9:30 AM
↩
Patch Set 1: Code-Review-1
(1 comment)
Modules/Core/Common/include/itkDomainThreader.hxx
Line 101:
This cout line needs to be deleted.

Dzenan Zukic
Patch Set 1: Code-Review+1 Looks good with the removal of the debugging output.9:39 AM

### CODE REVIEW

**Comment by dzenanz** [comment]
Why did you close this? I thought Brad wanted to give it a minor update and it would be ready for merging.

**Comment by hjmjohnson** [comment]
OOH!!!! I must have screwed up pushing items from gerrit to my account and accidentally  deleted all my local branches!

**Comment by dzenanz** [comment]
> accidentally deleted all my local branches!

Somebody else came to the same conclusion. Is there any good/quick way to un-delete the branches? And I am not sure that will automatically reopen the issues.

**Comment by hjmjohnson** [comment]
I did this manually.  It should be good now.

**Comment by hjmjohnson** [comment]
@blowekamp This should be ready soon.  Just waiting on the macOS build.

**Comment by blowekamp** [comment]
This patch should be ready, the build error:
https://open.cdash.org/viewBuildError.php?buildid=5616657
Seems to be a download issue.

File: Modules/Core/Common/include/itkDomainThreader.hxx [hunk 1]
```diff
@@ -92,28 +81,24 @@ void
...
-  else if( this->m_NumberOfWorkUnitsUsed > threaderNumberOfThreads )
+  this->GetMultiThreader()->SetNumberOfWorkUnits(this->m_NumberOfWorkUnitsUsed);
+
+  if( this->m_NumberOfWorkUnitsUsed > numberOfWorkUnits )
     {
     itkExceptionMacro( "A subclass of ThreadedDomainPartitioner::PartitionDomain"
                       << "returned more subdomains than were requested" );
     }
+
+  std::cout << "DomainThreader::DetermineNumberOfWorkUnits " << this->GetMultiThreader()->GetNumberOfWorkUnits() << std::endl;
```
[the remark this answers is not in the document]
Comment (hjmjohnson) [reply Modules/Core/Common/include/itkDomainThreader.hxx]: Fixed and pushed.

**Comment by hjmjohnson** [comment]
Let's see if the dashboard returns cleanly.

**Pull request merged**